src/rospkg/environment.py

```diff
@@ -109,7 +109,7 @@ def get_ros_package_path(env=None):
     """
     if env is None:
         env = os.environ
-    return env.get(ROS_PACKAGE_PATH, None)
+    return env.get(ROS_PACKAGE_PATH, '') + ':/usr/share'
```

Comment:
You could also just generate the normal catkin `setup.*sh` files and install them to something like `/usr/share/ros/setup.*sh`. Then to use ros you need to source this file, but that's not a huge burden, imo. Plenty of other software have a similar mechanism to setup the environment before being used. You could still source this file by default by using `/etc/profile.d` or similar.